stable/vault/README.md

```diff
@@ -26,7 +26,7 @@ To install the chart, use the following, this backs Vault with a Consul cluster:
 ```
 helm init -c; 
 helm repo add banzaicloud-stable http://kubernetes-charts.banzaicloud.com
-helm install banzaicloud-stable/vault --set vault.dev=false --set vault.config.storage.consul.address="myconsul-svc-name:8500",vault.config.storage.consul.path="vault"
+helm install banzaicloud-stable/vault --set vault.config.storage.consul.address="myconsul-svc-name:8500",vault.config.storage.consul.path="vault"
```

Comment:
Can you change the banzaicloud-stable repo url to `helm repo add banzaicloud-stable http://kubernetes-charts.banzaicloud.com/branch/master`?